windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc

```diff
@@ -0,0 +1,36 @@
+[id="creating-windows-machineset-vsphere"]
+= Creating a Windows MachineSet object on vSphere
+include::modules/common-attributes.adoc[]
+:context: creating-windows-machineset-vsphere
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on VMware vSphere. For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
+
+[discrete]
+== Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You are using a supported Windows Server as the operating system image with the Docker-formatted container runtime add-on enabled.
+
+[IMPORTANT]
+====
+Currently, the Docker-formatted container runtime is used in Windows nodes. Kubernetes is deprecating Docker as a container runtime; you can reference the Kubernetes documentation for more information in link:https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/[Docker deprecation]. Containerd will be the new supported container runtime for Windows nodes in a future release of Kubernetes.
+====
+
+include::modules/machine-api-overview.adoc[leveloffset=+1]
+
+[id="preparing-vsphere-for-windows-containers"]
+== Preparing your vSphere environment for Windows container workloads
+
+You must prepare your vSphere environment for Windows container workloads.
```

Comment:
This might be redundant information, but do you think it would be helpful to briefly say here how you prepare your vSphere environment for Windows container workloads? Like include, "... by creating the vSphere Windows VM golden image and enabling communication with the internal API server for the WMCO on vSphere." Or something to that effect?